usage/usage.go

```diff
@@ -0,0 +1,67 @@
+package usage
+
+import (
+	"github.com/akitasoftware/akita-libs/api_schema"
+	"github.com/c9s/goprocinfo/linux"
+	"github.com/pkg/errors"
+)
+
+const (
+	selfStatusFile = "/proc/self/status"
+	selfStatFile   = "/proc/self/stat"
+	allStatFile    = "/proc/stat"
+)
+
+var (
+	lastStat    *linux.ProcessStat
+	lastAllStat *linux.Stat
+)
+
+// Initialize state for computing CPU/memory usage for telemetry.
+func Init() {
+	// No need to check errors here. If /proc files aren't available,
+	// subsequent calls to Get() will return nil.
+	lastAllStat, _ = linux.ReadStat(allStatFile)
+	lastStat, _ = linux.ReadProcessStat(selfStatFile)
```

Comment:
Do we want to log and/or report errors to the back end?

Reply:
After some refactoring in 34420f65732975c6c23ccd1a9f7f69546958eb66, we'll log the first failure that occurs.  I don't think we need to report it to the backend -- the only error is if the /proc fs isn't available, which is expected for e.g. macOS.